runtime_scan/pkg/cloudinit/cloudinit_template.go

```diff
@@ -35,6 +35,7 @@ write_files:
       [Service]
       Type=oneshot
       WorkingDirectory=/opt/vmclarity
+      ExecStartPre=mkdir -p /var/opt/vmclarity
       ExecStartPre=docker pull {{ .ScannerImage }}
       ExecStart=docker run --rm --name %n --privileged \
           -v /opt/vmclarity:/opt/vmclarity \
```

Comment:
Need to volume mount /var/opt/vmclarity here otherwise it'll output inside the container

Reply:
Good catch. Fixed.